### ISSUE TITLE
Make Pending Pulls and Pushes Easier to See

### ISSUE THREAD

**Issue by kralle333** [ForkIssues/TrackerWin#1447]
It happens often that I am unaware that I have any pending pushes or pulls when using Fork, since the numbers showing the number of commits to pull/push is tiny.

In SourceTree it is very easy to see, since a ring of another color is on top of the push and pull buttons with the the total number of push/pulls.
![image](https://user-images.githubusercontent.com/5738476/152763328-bd25930f-d641-48ba-b124-94021f61ff0a.png)

Would it be possible to add something like that to Fork?

**Comment by DanPristupov** [comment]
https://github.com/fork-dev/TrackerWin/issues/402

**Comment by octavobr** [comment]
Please keep this configurable to keep the interface simple. Too many numbers everywhere is also not good.

**Comment by JeReT** [comment]
Yeah, actually the number is not so important. It is more important to know that there is something that should be pulled in general. So instead of a number, the icon could just change.
